ocs_ci/framework/main.py

```diff
@@ -94,10 +95,16 @@ def main(argv=None):
     init_ocsci_conf(arguments)
     pytest_logs_dir = utils.ocsci_log_path()
     utils.create_directory_path(framework.config.RUN['log_dir'])
+    launch_name = f"Tier run from {getuser()} RunId: {framework.config.RUN['run_id']}"
```

Comment:
We don't know if this is a Tier run or a simple deployment so I think using a launch name like this could be misleading. Furthermore if we include the `run_id` in the launch name we won't be able to utilize the history feature in report portal. The launch name should be the same as other executions of the same nature. Report portal will append a number to the launch name as we execute several launches with the same name. This allows us to view the history and compare results from iteration to iteration.

I can see us writing a function that will use several items from the configuration and from the provided arguments in order to determine exactly what kind of deployment/tier test this is and then choose the correct launch name to provide to `--rp-launch`

Reply:
This is just an initial version, I want to have something similar to what our email subject says but that part of the code is all across repo.  If you have the code handy send me :)

Reply:
I don't have anything handy at the moment. I just wanted to raise this point early as it's fairly important to being able to fully leverage report portal.